bindep.txt

```diff
@@ -3,3 +3,5 @@
 
 git
 python38-wheel [platform:centos-8 platform:rhel-8]
+tox [test platform:dpkg]
+python3-pip [test platform:dpkg]
```

Comment:
This shouldn't be necessary. This file probably needs to be removed from the repo (not in this change).

Reply:
OK I will remove it but I find it quite useful for other projects. There is also the tox-extra plugin (which I maintain) which checks for bindep before running tox, saving time for first time contribs.